mhs/outbound/outbound/request/synchronous/handler.py

```diff
@@ -99,7 +100,11 @@ def _write_response(self, status: int, message: str) -> None:
         """
         logger.info('0010', 'Returning response with {HttpStatus}', {'HttpStatus': status})
         self.set_status(status)
-        self.set_header("Content-Type", "text/xml")
+        if 400 <= status:
+            content_type = "text/plain"
+        else:
+            content_type = "text/xml"
+        self.set_header("Content-Type", content_type)
```

Comment:
This is a bit confusing atm, I'm starting to think that I maybe should change the Tornado errors to be `text/plain` so that then all errors are at least consistent in terms of content type. The success responses, I presumed they'd be xml when writing the outbound handler. The acks would be (or they'd be empty), the responses might be, I'm not sure about multipart (and I'm not sure if that would be sent down the async channel or as a HTTP response, but at that time, I think I can change the code accordingly if necessary.